### PR TITLE
adding resource view to calendar

### PR DESCRIPTION
**Pull request by jeppekroghitk**

### CODE REVIEW

File: assets/src/components/calendar-cell-info-button.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import * as PropTypes from "prop-types";
+import "./calendar-cell-info-button.scss";
+
+/**
+ * @param root0
+ * @param root0.title
+ * @param root0.resourceId
+ * @param root0.onClickEvent
+ */
```
Comment (martinyde) [inline assets/src/components/calendar-cell-info-button.js]: This doesnt look right

File: assets/src/components/resource-view.js [hunk 2]
```diff
@@ -3,21 +3,69 @@ import * as PropTypes from "prop-types";
 import ResourceDetails from "./resource-details";
 import "./resource-view.scss";
 
+/**
```
Comment (martinyde) [inline assets/src/components/resource-view.js:6]: This doesnt look right

**Review by martinyde — CHANGES_REQUESTED** [review]

File: assets/src/components/calendar-cell-info-button.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import * as PropTypes from "prop-types";
+import "./calendar-cell-info-button.scss";
+
+/**
+ * @param root0
```
Comment (tuj) [inline assets/src/components/calendar-cell-info-button.js]: remove

File: assets/src/app.js [hunk 4]
```diff
@@ -28,8 +28,14 @@ function App() {
   const [authorFields, setAuthorFields] = useState({});
   // TODO: Add these.
   // const [resource, setResource] = useState(null);
+  const [resource, setResource] = useState(null);
+  const [facilities, setFacilities] = useState(null);
   // const [minimumSeatsRequired, setMinimumSeatsRequired] = useState(null);
 
+  // ResourceView overlay trigger & global data
+  const [showResourceView, setShowResourceView] = useState(null);
```
Comment (tuj) [inline assets/src/app.js]: Can these two states (showResourceView , resourceId) be merged to 

```
const [showResourceViewId, setShowResourceViewId] = useState(null);
```

and then you can deduct from whether or not showResourceViewId is null, whether or not you should show the ResourceView

**Review by tuj — CHANGES_REQUESTED** [review]

**Review by martinyde — APPROVED** [review]

**Pull request merged**